### DISCUSSION TITLE
Chat reply length limit

### DISCUSSION THREAD

**Discussion by fedorbass** [brianpetro/obsidian-smart-connections#343] [Q&A] [answered]
Is there a maximum character limit for the response? I asked for a long note to be rewritten, but the reply was cut off at the 1,327th character. Using a 16k model in settings.

**Comment by fedorbass** [comment]
Found "max_available_tokens: 200" in the developer console.
And the reply is roughly 200 tokens long. Do I have to modify it somewhere?

**Reply by RealRavens** [reply, marked as answer]
Tokens to prompt and respond use the same pool of tokens. So I guess the response just wasn't enough to make it. One way around it is let's say you're summarizing, if a response is cut off, you could ask the AI to continue and it will in the next response. You could also split your text a little bit, again assuming it's a summary and then give it more text and say continue summary with this text blah blah whatever your prompt is. Hope that helps a bit.

**Reply by fedorbass** [reply]
Thanks! The "Continue" reply worked.

**Reply by RealRavens** [reply]
Anytime my friend~ 👍
